### PR TITLE
Improve Dolby Millicast APIs

### PR DESCRIPTION
**Pull request by FabienLavocat**
**Breaking Changes**
- Remove Communications APIs
- Rename some classes in Dolby Millicast APIs

**Changes**
- Use dataclasses from Python to parse JSON objects
- Add `account` and `webhooks` Millicast APIs
- Update GitHub workflows

### CODE REVIEW

File: client/src/dolbyio_rest_apis/streaming/models/cluster.py [hunk 1]
```diff
@@ -2,28 +2,34 @@
...
+    """The :class:`Cluster` object, which represents a cluster."""
 
-        self.id = get_value_or_default(self, 'id', None)
-        self.name = get_value_or_default(self, 'name', None)
-        self.rtmp = get_value_or_default(self, 'rtmp', None)
+    id: str
+    name: str
+    rtmp: str
+    srt: str
+    location: ClusterLocation
```
Comment (jchang0419) [inline client/src/dolbyio_rest_apis/streaming/models/cluster.py:33]: @FabienLavocat missing the features obj
https://docs.dolby.io/streaming-apis/reference/cluster_getclustersinfo

File: client/src/dolbyio_rest_apis/streaming/models/webhooks.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    id: int | None = None
+    url: str | None = None
+    secret: str | None = None
+    is_feed_hooks: bool | None = None
+    is_recording_hooks: bool | None = None
+    is_thumbnail_hooks: bool | None = None
+    is_transcoder_hooks: bool | None = None
+    is_clip_hooks: bool | None = None
+
+class UpdateWebhook:
```
Comment (jchang0419) [inline client/src/dolbyio_rest_apis/streaming/models/webhooks.py:25]: missing refreshSecret property 
refreshSecret boolean | null
If true will generate new signing secret for webhook.
https://docs.dolby.io/streaming-apis/reference/webhooks_updatewebhook

File: client/src/dolbyio_rest_apis/streaming/models/subscribe_token.py [hunk 3]
```diff
@@ -2,64 +2,47 @@
...
-    def __init__(self, dictionary: dict):
-        dict.__init__(self, dictionary)
-
-        self.stream_name = get_value_or_default(self, 'streamName', None)
-        self.is_regex = get_value_or_default(self, 'isRegex', False)
-
-class SubscribeToken(dict):
+@dataclass_json(letter_case=LetterCase.CAMEL)
+@dataclass
+class SubscribeToken:
```
Comment (jchang0419) [inline client/src/dolbyio_rest_apis/streaming/models/subscribe_token.py:21]: missing tracking property returned in read token api
https://docs.dolby.io/streaming-apis/reference/subscribetokenv1_readtoken

File: client/src/dolbyio_rest_apis/streaming/webhooks.py [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+        api_secret: str,
+        webhook_id: int,
+    ) -> None:
+    async with StreamingHttpContext() as http_context:
+        await http_context.requests_delete(
+            api_secret=api_secret,
+            url=f'{get_rts_url()}/api/webhooks/{webhook_id}',
+        )
+
+async def update(
```
Comment (jchang0419) [inline client/src/dolbyio_rest_apis/streaming/webhooks.py:35]: missing refreshSecret param
https://docs.dolby.io/streaming-apis/reference/webhooks_updatewebhook

**Review by jchang0419 — CHANGES_REQUESTED** [review]
a couple of props missing here and there

File: client/src/dolbyio_rest_apis/streaming/models/cluster.py (hunk 1 above)
Comment (FabienLavocat) [reply client/src/dolbyio_rest_apis/streaming/models/cluster.py:33]: ✅

File: client/src/dolbyio_rest_apis/streaming/models/webhooks.py (hunk 2 above)
Comment (FabienLavocat) [reply client/src/dolbyio_rest_apis/streaming/models/webhooks.py:25]: ✅

File: client/src/dolbyio_rest_apis/streaming/models/subscribe_token.py (hunk 3 above)
Comment (FabienLavocat) [reply client/src/dolbyio_rest_apis/streaming/models/subscribe_token.py:21]: ✅

File: client/src/dolbyio_rest_apis/streaming/webhooks.py (hunk 4 above)
Comment (FabienLavocat) [reply client/src/dolbyio_rest_apis/streaming/webhooks.py:35]: ✅

**Review by jchang0419 — APPROVED** [review]

**Pull request merged**